packages/connector-google/src/index.ts

```diff
@@ -80,7 +80,10 @@ export const getAccessToken = async (
 
   const { access_token: accessToken } = result.data;
 
-  assert(accessToken, new ConnectorError(ConnectorErrorCodes.SocialAuthCodeInvalid));
+  assert(
+    accessToken,
+    new ConnectorError(ConnectorErrorCodes.SocialAuthCodeInvalid, 'accessToken is missing.')
+  );
```

Comment:
actually the access token logic is very repetitive as far as i can see. should be extracted into a function?

Reply:
Yes, this check is necessary for almost all social connectors. We can extract to @logto/connector-kit and seems it is not a blocker of GA.

Reply:
created issue?

Reply:
Check LOG-5808